.devcontainer/minimal.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:23.10
+FROM --platform=linux/amd64  ubuntu:23.10
```

Comment:
We really shouldn't do this... This is super slow on Linux (if you even have qemu-user-static installed).